benches/complex.rs

```diff
@@ -1,6 +1,6 @@
 use criterion::{criterion_group, criterion_main, Criterion};
 
-fn build_deep_hierarchy(stretch: &mut sprawl::node::Stretch) -> sprawl::node::Node {
+fn build_deep_hierarchy(stretch: &mut sprawl::node::Sprawl) -> sprawl::node::Node {
```

Comment:
I think if we do this, we also need to rename the variables.
```suggestion
fn build_deep_hierarchy(sprawl: &mut sprawl::node::Sprawl) -> sprawl::node::Node {
```

Reply:
Agreed; we should rename all the `stretch` variables as part of this PR.